doc/Makefile.am

```diff
@@ -2,8 +2,6 @@
 # Makefile.am for .../agent-ldap/doc
 #
 
-SUBDIRS = autodocs
```

Comment:
I just disable it for building in `make`, but I would like to keep it if someone want to intentionally generate it with doxygen on his own machine